lgc/elfLinker/ElfLinker.cpp

```diff
@@ -948,29 +948,29 @@ void OutputSection::layout() {
     }
 
     // Gain alignment as required for the next input section.
-    uint64_t alignment = getAlignment(inputSection);
+    Align alignment = getAlignment(inputSection);
     m_alignment = std::max(m_alignment, alignment);
-    size = (size + alignment - 1) & -alignment;
+    size = alignTo(size, alignment);
     // Store the start offset for the section.
     inputSection.offset = size;
     // Add on the size for this section.
     size += inputSection.size;
   }
   if (m_type == ELF::SHT_NOTE)
-    m_alignment = 4;
+    m_alignment = Align(4);
 }
 
 // =====================================================================================================================
 // Get alignment for an input section. This takes into account the reduceAlign flag, reducing the alignment
 // from 0x100 to 0x40 when gluing code together.
 //
 // @param inputSection : InputSection
-uint64_t OutputSection::getAlignment(const InputSection &inputSection) {
-  uint64_t alignment = inputSection.sectionRef.getAlignment();
+Align OutputSection::getAlignment(const InputSection &inputSection) {
+  Align alignment = Align(inputSection.sectionRef.getAlignment());
```

Comment:
`getAlignment` should already return an `Align` now?

Reply:
Yes but this makes it also work with LLVM without D139110, when getAlignment returned uint64_t.